frontend/src/components/doctors-filtration-panel/doctors-filtration-panel.tsx

```diff
@@ -0,0 +1,126 @@
+    defaultValues: DEFAULT_VALUES,
+    mode: "onChange"
+  });
+
+  const handleSubmitForm = (formData: IDoctorFiltrationPayload) => {
+    console.log(formData);
+  };
+
+  return (
+    <form onSubmit={handleSubmit(handleSubmitForm)}>
+    <div className={styles.panel}>
+      <div className={styles.filters}>
+        <div className={styles.filterHeader}>Filter by</div>
+        <div className={styles.commonFilter}>
+          <TextInput
+            name={DoctorFiltration.DOCTOR_NAME}
+            label=""
+            hasHiddenLabel={false}
+            placeholder="Type a doctor name..."
+            type={InputType.SEARCH}
+            color={InputColor.WHITE}
+            control={control}
+            errors={errors}
+            />
+        </div>
+        <Details
+          icon={location}
+          title="location"
+        >
+          <TextInput
+            name={DoctorFiltration.CITY}
+            label=""
+            hasHiddenLabel={false}
+            placeholder="City..."
+            type={InputType.TEXT}
+            color={InputColor.WHITE}
+            control={control}
+            errors={errors}
+          />
+        </Details>
+        <Details
+          icon={specialty}
+          title={DoctorFiltration.SPECIALTY}
+        >
+          <div className={styles.filterCheckbox}>
+            <Checkbox
+              name={DoctorType.PEDIATRICIAN}
+              label={DoctorType.PEDIATRICIAN}
+              control={control}
+            />
+          </div>
+            <div className={styles.filterCheckbox}>
+            <Checkbox
+              name={DoctorType.ENDOCRINOLOGIST}
+              label={DoctorType.ENDOCRINOLOGIST}
+              control={control}
+            />
+          </div>
+            <div className={styles.filterCheckbox}>
+            <Checkbox
+              name={DoctorType.DENTIST}
+              label={DoctorType.DENTIST}
+              control={control}
+            />
+          </div>
+            <div className={styles.filterCheckbox}>
+            <Checkbox
+              name={DoctorType.SURGEON}
+              label={DoctorType.SURGEON}
+              control={control}
+            />
+          </div>
+          <div className={styles.filterCheckbox}>
+          <Checkbox
+              name={DoctorType.DERMATOLOGIST}
+              label={DoctorType.DERMATOLOGIST}
+              control={control}
+            />
+          </div>
+        </Details>
+        <Details
+          icon={clinic}
+          title={DoctorFiltration.TYPE_OF_CLINIC}
+        >
+          <div className={styles.filterCheckbox}>
+            <Checkbox
+              name={ClinicType.PRIVATE}
+              label={ClinicType.PRIVATE}
+              control={control}
+            />
+          </div>
+            <div className={styles.filterCheckbox}>
+            <Checkbox
+              name={ClinicType.STATE}
+              label={ClinicType.STATE}
+              control={control}
+            />
```

Comment:
```suggestion
     const clinicTypes = Object.keys(ClinicType)


     {clinicTypes.map((clinicType) => (
            <Checkbox
              name={clinicType}
              label={clinicType}
              control={control}
            />
     ))}
```

Also do the same for checkboxes